README.md

```diff
@@ -1 +1,46 @@
-# StackOverflow-lite
\ No newline at end of file
+[![Build Status](https://travis-ci.org/emukungu/StackOverflow-lite.svg?branch=ft-post-a-question-%23159866594)](https://travis-ci.org/emukungu/StackOverflow-lite) [![Coverage Status](https://coveralls.io/repos/github/emukungu/StackOverflow-lite/badge.svg?branch=ft-post-a-question-%23159866594)](https://coveralls.io/github/emukungu/StackOverflow-lite?branch=ft-post-a-question-%23159866594) [![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)](https://codeclimate.com/github/codeclimate/codeclimate/test_coverage)
+
+# StackOverflow-lite
+StackOverflow-lite is a platform where people can ask questions and provide answers.
+
+## Project APIs
+
+API endpoint | Functionality
+-------------|--------------
+GET /api/v1/questions| Get all questions
+POST /api/v1/questions| Post a question
+GET /api/v1/questions/questionId| Get a specific question
+POST api/v1/questions/questionId/answer| Post an answer
+
+## Setup/Installation Requirements
+* git bash
```

Comment:
I don't think that these are the correct requirements for users to install your application because a person can setup your application using any other command line client, text editor/IDE and API test client. Since this is a python project, Python should be one of the requirements and it would be better to specify the exact python version

Reply:
Have removed the specific requirements to allow for any other requirements as desired by the developer.